packages/dds/merge-tree/tsconfig.json

```diff
@@ -7,6 +7,7 @@
         "rootDir": "./src",
         "outDir": "./dist",
         "composite": true,
+        "downlevelIteration": true,
```

Comment:
Did we run into something where this is relevant, or is it preemptive?

Reply:
Weird. I had thought this was necessary due to a compiler error when pulling things out to an iterator--which makes sense at a glance--but closer review of the spec shows it should only be necessary when targeting ES3/ES5, and we're past those standards. Removing it seems to work fine on my local build now, so reverted. (Not sure what was up with that). Thanks for questioning.